packages/blockchain-wallet-v4-frontend/src/scenes/Airdrops/StxAirdrop/model.tsx

```diff
@@ -27,10 +28,10 @@ const BlueCartridge = styled(CustomCartridge)`
 // userCampaignState: "NONE", "REGISTERED", "TASK_FINISHED", "REWARD_SEND", "REWARD_RECEIVED", "FAILED"
 // userCampaignTransactionResponseList: []
 export const StxStatus = ({
-  userCampaignsInfoResponseList = [],
+  userCampaignsInfoResponseList,
```

Comment:
do we not need to default to an array anymore?

Reply:
Nope, because of line 61